src/packages/excalidraw/CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ The change should be grouped under one of the below section and must contain PR
 Please add the latest change on the top under the correct section.
 -->
 
+## Unreleased
+
+## Excalidraw Library
+
+#### Breaking change
```

Comment:
Shouldn't this be a fix instead of breaking change ? 
& also we might want to move it into `Excalidraw API` since it will impact the hosts ? 
Excalidraw library is the list of updates pushed to excalidraw chore so this might get lost in that list.

Reply:
How this is Excalidraw API tho since it doesn't affect the API, and is specific to the core. Not sure where we draw the line though.

> Shouldn't this be a fix instead of breaking change ?

I guess. I thought it might break some host apps where they may have counted on the breakpoint behaving as it did and it'll break their UI in some way. https://xkcd.com/1172/